### PR TITLE
Add mac support (with warning that docking scores might not match)

### PR DESCRIPTION
**Pull request by AustinT**
This PR includes a version of Vina for Mac, its license, and a warning that docking scores might not match.

Would solve issue #141

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,11 +12,9 @@ and our [website](https://dockstring.github.io/):
 
 ## Installation
 
-Currently this package is **only supported on Linux**,
-and we do not plan to add support for Mac/Windows.
-See [this issue](https://github.com/dockstring/dockstring/issues/141)
-for more information on why this is
-(feel free to re-open it if you want to help us provide Mac/Windows support).
+This package is primarily intended for Linux, but we have some support for Mac.
```
Comment (mgarort) [inline README.md:15]: I would change

`However, the scores from the Mac version does not match the Linux version,
so we encourage the use of the Linux version whenever possible.`

for 

`Please note that the scores from the Mac version do not always perfectly match the Linux version, so we encourage the use of the Linux version whenever possible.`

File: README.md [hunk 2]
```diff
@@ -43,6 +41,15 @@ As such, we recommend you install it in the following way.
    ```bash
    python tutorials/simple_example.py
    ```
+1. *(optional)* Check whether your local version of dockstring matches the dockstring dataset.
+   This is only necessary if you plan to mix docking scores from the dockstring dataset
```
Comment (mgarort) [inline README.md:45]: I would add

`This is only if you plan to mix PREVIOUSLY-COMPUTED docking scores from the docking dataset`

File: README.md [hunk 3]
```diff
@@ -43,6 +41,15 @@ As such, we recommend you install it in the following way.
...
+   with locally-computed scores, or if you want to compare results with the dockstring paper.
+   We have created a `pytest` test for this. To run it:
+   ```bash
+   conda install -c conda-forge pytest  # only if not installed already
+   python -m pytest tests/test_dataset_matching.py
+   ```
+   If the test passes then your local version of docktring matches the dataset! 🥳
```
Comment (mgarort) [inline README.md:52]: `If the test passes then your local version of dockstring matches the dataset EXACTLY!`

File: dockstring/resources/bin/LICENSE
Comment (mgarort) [inline dockstring/resources/bin/LICENSE:1]: @gncs I think you added this license. Why do we need it? Is it not the same Apache license that we already had?

From what I understood, Vina's license is Apache.

File: dockstring/utils.py [hunk 4]
```diff
@@ -63,6 +63,13 @@ def get_vina_filename() -> str:
     system_name = platform.system()
     if system_name == 'Linux':
         return 'vina_linux'
+    if system_name == 'Darwin':
+        warnings.warn(
+            "Although Mac use is supported, docking scores on Mac generally *do not* "
```
Comment (mgarort) [inline dockstring/utils.py:68]: `"Although Mac use is supported, docking scores on Mac do not always perfectly match scores from Linux. Therefore, extra care should be taken when comparing results to other platforms. In particular, the baselines in the DOCKSTRING paper were computed on Linux, so please do not directly compare your docking scores to the scores reported on the paper."

**Review by mgarort — COMMENTED** [review]
I have suggested some modifications to make the wording more positive.

File: dockstring/resources/bin/LICENSE
Comment (AustinT) [reply dockstring/resources/bin/LICENSE:1]: I guess it doesn't hurt to include an extra copy of the license?

**Pull request merged**